### DISCUSSION TITLE
Option to use Long-integer Versions of Cuba rountines

### DISCUSSION THREAD

**Discussion by kmbrune** [secdec-research/secdec#15] [Q&A] [answered]
Hello,

I was wondering if there is a quick way of using the Long-integer Versions of Cuba inside pySecDec ?

Thanks, 
Kevin

**Comment by spj101** [comment, marked as answer]
Hi @kmbrune,

By default we use only the `long long int` versions of Cuba (`llVegas`, `llCuhre`, ...). See `pySecDecContrib/util/secdecutil/integrators/cuba.hpp`.

Did you encounter an issue with the use of Cuba that prompted your question?
